mouse/record.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import print_function
+
+__author__ = "Bailey Nozomu Hwa <hwab@janelia.hhmi.org"
+__date__ = "$July 30, 2015 20:20:18 EDT$"
+
+import argparse
+import io
+import time
+import picamera
+import datetime
+import os
+
+
+def main(*argv):
+    """
+        Simple main function. Defines arguments: time before and after the initiation of a trigger event,
```

Comment:
This line is probably too long. I would probably just drop "Defines arguments: ..." to a newline. Also, might change "Simple main function." -> to "Simple main function that performs event center recording." or something like this. Feel free to play with the wording.